js/src/hooks/useDetailedErrorBySlots.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * External dependencies
+ */
+import { useSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { STORE_KEY } from '~/data/constants';
+
+const selectorName = 'getDetailedErrorBySlots';
+
+const useDetailedErrorBySlots = ( slots ) => {
```

Comment:
@ankitrox Can we add some comment explaining what the hook does. Specifically that the first error from the slot is rendered.